challenge.md

```diff
@@ -29,19 +29,19 @@ Open up this file in Atom, and write your answers below (where indicated).
 
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
- <!-- Answer Starts Here -->
+ <!-- Answer Starts Here --> This is an absolute path. It starts with a '/'
 
  <!-- Answer Ends Here -->
 
 2. Suppose that we're working on a project, and we want to use a font that's being hosted somewhere on the internet. Would we use an absolute or relative path to refer to it? Why?
 
- <!-- Answer Starts Here -->
+ <!-- Answer Starts Here -->Absolute path
 
  <!-- Answer Ends Here -->
 
 3. Now suppose that we have an image file living inside our project. Would we refer to it with an absolute or relative path? Why?
 
- <!-- Answer Starts Here -->
+ <!-- Answer Starts Here -->You can use either but absolute is recommended because it requires less maintence when updating your file and is able to use throughout.
```

Comment:
relative path would be what you would use here because your going from the file you want the image in to the file where the image lives